### PR TITLE
fix: map makers identity check

### PR DESCRIPTION
**Pull request by manuel-calavera**

### CODE REVIEW

File: src/v2/stores/nodes.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ class Store {
           pubkey,
           gossip,
           coordinates,
-          name: (identity && identity.name) || identity.pubkey,
+          name: (identity && (identity.name || identity.pubkey)) || '',
```
Comment (sunnygleason) [inline src/v2/stores/nodes.js]: I think it should be:

`(identity && identity.name) || pubkey || ''`

Comment (manuel-calavera) [reply src/v2/stores/nodes.js]: if you insist, both are ok. identity has pubkey field

**Comment by sunnygleason** [comment]
@manuel-calavera thank you! merging...

**Pull request merged**